thorlcr/thorutil/thormisc.cpp

```diff
@@ -92,6 +92,7 @@ const StatisticsMapping graphStatistics({StNumExecutions, StSizeSpillFile, StSiz
 const StatisticsMapping diskReadPartStatistics({StNumDiskRowsRead}, diskReadRemoteStatistics);
 const StatisticsMapping indexDistribActivityStatistics({}, basicActivityStatistics, jhtreeCacheStatistics);
 const StatisticsMapping soapcallActivityStatistics({}, basicActivityStatistics, soapcallStatistics);
+const StatisticsMapping hashDedupActivityStatistics({StNumSpills, StSizeSpillFile}, diskWriteRemoteStatistics, basicActivityStatistics);
```

Comment:
spillStatistics is used for both sorting activities and join activities. So, ideally I would need to use the new SpillStatistics for keyed_join and sortedSpillStatistics for sorting activities.  It does make more sense but
1) it will require changes to 6 files (12 lines of code)
2) the use of SpillStatistics for join activities is a small change in behaviour unrelated to this change.

I'm not sure this change should be in the scope of this PR.

Reply:
>use the new SpillStatistics for keyed_join and sortedSpillStatistics for sorting activities

Yes.

>I'm not sure this change should be in the scope of this PR. 

In 2 minds, but yes with the aim of making PR's turnaround quicker/smaller etc. I agree.
But I'd add the StTimeSortElapsed change into dedup stats and impl. now, as it's part of the same issue.

Then a separate PR after to do the sortedSpillStatistics change.